tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_bug_1116566.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import SkipTest
+from marionette.by import By
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.email.app import Email
+from gaiatest.apps.system.regions.cards_view import CardsView
+
+
+class TestBug1116566(GaiaTestCase):
+    _menu_button_locator = (By.CSS_SELECTOR, '.msg-folder-list-btn')
```

Comment:
Ok, sounds good. I don't really know what PageRegion is doing, currently, to be hones.